### ISSUE TITLE
Revamp the UI

### ISSUE THREAD

**Issue by vaibhav-sharma-frontend** [arteevraina/Music-App#6]
The current UI of the web app is unwelcoming so design the new UI of the app to give it a new feel with a new color scheme.
You can use Adobe XD, Figma or sketch any software for designing.

Once the design is ready you share your work then we all can share our view on the new design and start implementing it.
For some inspirations visit: https://dribbble.com/tags/music_player

**Comment by vaibhav-sharma-frontend** [comment]
@arteevraina what's your opinion on this buddy?

**Comment by arteevraina** [comment]
@vsharma20v  yes the UI must be improved

**Comment by Nishantgoyal918** [comment]
I want to work on this issue

**Comment by ShrinivasSK** [comment]
There is an issue with the UI. We can scroll down the main page and see a black region below. 
I don't know how to correct this so I am posting it here.
![Screenshot from 2019-12-08 04-02-37](https://user-images.githubusercontent.com/56220091/70378034-c2345680-1941-11ea-98cd-f76e9563560f.png)

**Comment by vaibhav-sharma-frontend** [comment]
> There is an issue with the UI. We can scroll down the main page and see a black region below.
> I don't know how to correct this so I am posting it here.
> ![Screenshot from 2019-12-08 04-02-37](https://user-images.githubusercontent.com/56220091/70378034-c2345680-1941-11ea-98cd-f76e9563560f.png)

Feel free to create a new issue for this problem

**Comment by vaibhav-sharma-frontend** [comment]
@Nishantgoyal918 go for it
